{"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"MapperError"}],"languages":["swift"],"platforms":["iOS"]}]},{"kind":"content","content":[{"anchor":"overview","level":2,"type":"heading","text":"Overview"},{"type":"unorderedList","items":[{"content":[{"type":"paragraph","inlineContent":[{"type":"text","text":"dataDecodingWithError: Unsuccessful data decoding."}]}]}]}]}],"schemaVersion":{"major":0,"minor":3,"patch":0},"sections":[],"variants":[{"paths":["\/documentation\/appcraft_auth_ios\/acautherror\/mappererror"],"traits":[{"interfaceLanguage":"swift"}]}],"relationshipsSections":[{"identifiers":["doc:\/\/appcraft-auth-ios\/10Foundation14LocalizedErrorP","doc:\/\/appcraft-auth-ios\/s5ErrorP","doc:\/\/appcraft-auth-ios\/s8SendableP"],"kind":"relationships","title":"Conforms To","type":"conformsTo"}],"identifier":{"url":"doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError","interfaceLanguage":"swift"},"abstract":[{"type":"text","text":"Represents the error reason during data decoding phase."}],"kind":"symbol","metadata":{"fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"MapperError"}],"title":"ACAuthError.MapperError","roleHeading":"Enumeration","role":"symbol","symbolKind":"enum","externalID":"s:17appcraft_auth_ios11ACAuthErrorO06MapperE0O","modules":[{"name":"appcraft-auth-ios"}],"navigatorTitle":[{"kind":"identifier","text":"MapperError"}]},"hierarchy":{"paths":[["doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios","doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError"]]},"topicSections":[{"title":"Enumeration Cases","identifiers":["doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError\/dataDecodingWithError(_:)"]},{"title":"Default Implementations","identifiers":["doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError\/Error-Implementations","doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError\/LocalizedError-Implementations"],"generated":true}],"references":{"doc://appcraft-auth-ios/documentation/appcraft_auth_ios/ACAuthError":{"role":"symbol","title":"ACAuthError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ACAuthError"}],"abstract":[],"identifier":"doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"ACAuthError"}],"url":"\/documentation\/appcraft_auth_ios\/acautherror"},"doc://appcraft-auth-ios/documentation/appcraft_auth_ios/ACAuthError/MapperError/dataDecodingWithError(_:)":{"role":"symbol","title":"ACAuthError.MapperError.dataDecodingWithError(_:)","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"dataDecodingWithError"},{"kind":"text","text":"("},{"kind":"typeIdentifier","text":"Error","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":"?)"}],"abstract":[{"type":"text","text":"Unsuccessful data decoding"}],"identifier":"doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError\/dataDecodingWithError(_:)","kind":"symbol","type":"topic","url":"\/documentation\/appcraft_auth_ios\/acautherror\/mappererror\/datadecodingwitherror(_:)"},"doc://appcraft-auth-ios/documentation/appcraft_auth_ios/ACAuthError/MapperError":{"role":"symbol","title":"ACAuthError.MapperError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"MapperError"}],"abstract":[{"type":"text","text":"Represents the error reason during data decoding phase."}],"identifier":"doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError","kind":"symbol","type":"topic","navigatorTitle":[{"kind":"identifier","text":"MapperError"}],"url":"\/documentation\/appcraft_auth_ios\/acautherror\/mappererror"},"doc://appcraft-auth-ios/documentation/appcraft_auth_ios":{"role":"collection","title":"appcraft-auth-ios","abstract":[],"identifier":"doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios","kind":"symbol","type":"topic","url":"\/documentation\/appcraft_auth_ios"},"doc://appcraft-auth-ios/s8SendableP":{"type":"unresolvable","title":"Swift.Sendable","identifier":"doc:\/\/appcraft-auth-ios\/s8SendableP"},"doc://appcraft-auth-ios/10Foundation14LocalizedErrorP":{"type":"unresolvable","title":"Foundation.LocalizedError","identifier":"doc:\/\/appcraft-auth-ios\/10Foundation14LocalizedErrorP"},"doc://appcraft-auth-ios/documentation/appcraft_auth_ios/ACAuthError/MapperError/Error-Implementations":{"role":"collectionGroup","title":"Error Implementations","abstract":[],"identifier":"doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError\/Error-Implementations","kind":"article","type":"topic","url":"\/documentation\/appcraft_auth_ios\/acautherror\/mappererror\/error-implementations"},"doc://appcraft-auth-ios/documentation/appcraft_auth_ios/ACAuthError/MapperError/LocalizedError-Implementations":{"role":"collectionGroup","title":"LocalizedError Implementations","abstract":[],"identifier":"doc:\/\/appcraft-auth-ios\/documentation\/appcraft_auth_ios\/ACAuthError\/MapperError\/LocalizedError-Implementations","kind":"article","type":"topic","url":"\/documentation\/appcraft_auth_ios\/acautherror\/mappererror\/localizederror-implementations"},"doc://appcraft-auth-ios/s5ErrorP":{"type":"unresolvable","title":"Swift.Error","identifier":"doc:\/\/appcraft-auth-ios\/s5ErrorP"}}}